app/models/article.rb

```diff
@@ -72,6 +72,8 @@ class Article < ApplicationRecord
   validate :validate_collection_permission
   validate :validate_tag
   validate :validate_video
+  validate :validate_co_authors
+  validate :validate_co_authors_must_not_be_the_same
```

Comment:
I thought about it but I'd somehow liked the idea of preserving insertion order so that "co-authored by alice and bob" would always be in this order, even if another alice is added at the end

Reply:
@rhymes when you say "preserving insertion order" are you hoping that the names are always alphabetically ordered even if the co-author names are not added in alphabetical order (Bob is added as a co-author before Alice is, but Alice's name appears first on the article) or are you hoping for something else? 😄

Reply:
No, not the alphabetical order. If today I add Bob and tomorrow I add Alice, being this an array, it should stay "bob and alice", never "alice and bob". But I don't feel strongly about this, whatever you choose is fine :)

Reply:
Ahh, I see! Thank you for the clarification @rhymes. 😄